dspace-api/pom.xml

```diff
@@ -354,6 +354,10 @@
             <groupId>commons-collections</groupId>
             <artifactId>commons-collections</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
```

Comment:
While I don't object to it, I'd like to ensure both of these versions of `commons-collections` are necessary. I notice we already have it just above, and this PR adds in `commons-collections4`.

Reply:
Collections4 offers some new calls like `emptyIfNull`. I agree that we should upgrade all our `commons-collections` usages to version 4, but I think we should do that in a separate ticket.

Reply:
I've created this ticket to address this: https://jira.duraspace.org/browse/DS-3782